lib/pkgcloud/azure-v2/compute/client/servers.js

```diff
@@ -27,7 +27,9 @@ function getVersion(callback) {
  */
 function getLimits(callback) {
   return errs.handle(
-    errs.create({ message: 'Azure\'s API is not rate limited' }),
+    errs.create({
+      message: 'Azure\'s API is not rate limited'
+    }),
```

Comment:
it is beautify - sorry about that, but as i said the real changes are very small. I will disable it for future use on this project